### ISSUE TITLE
Multi-account : When the client starts, the first account in the account list is always selected (main dialog), without taking into account the previously selected account

### ISSUE THREAD

**Issue by Jerome-Herbinet** [nextcloud/desktop#5523]
### Problem description (and solution)

**Context :** When the client is connected to several accounts and this one has just started, then the main dialog is opened, the first account in the list is always selected. 

**The problem** is that (for whatever reason), the account I use the most is not the first one in the list of accounts (the list I can access by clicking on my name at the top of the main dialog), as soon as I open the main dialog, the account that appears is the wrong one. 

**In my opinion**, this needs to be improved, not only for the sake of convenience, but also because it is more relevant for multi-account use.

**Below**, imagine that the account I use the most (for whatever reason, it is my right and my need) is the one I have framed in green. The problem is that after starting or restarting the application (in 99% of the cases when I turn on my computer in the morning to work), it is the first account in the list that is systematically selected (the one I have framed in red) ... and so, when I open the main dialog, I am obliged to switch to the right account. This is not practical. 

![2023-03-14_09-35](https://user-images.githubusercontent.com/33763786/224942911-4136978e-28cc-4ad9-947d-f5c78647048c.png)

**To summarize**, when the synchronization client starts, the last selected account should be selected when opening the main dialog.

**Comment by Rello** [comment]
Commit [fb9fd945](https://github.com/nextcloud/desktop/commit/fb9fd945a0) stores and restores the selected account
